core/src/main/scala/org/apache/gearpump/jarstore/JarFileContainerWrapper.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.gearpump.jarstore
+
+import java.io._
+
+import org.apache.gearpump.cluster.UserConfig
+import JarFileContainerWrapper._
+
+/**
```

Comment:
I think the JarFileContainerWrapper itself is clear. No need to add class comments in the class definition of JarFileContainerWrapper.

We only need to explain why we use JarFileContainerWrapper at where it is used. Like JarStoreService.